catalog-info.yaml

```diff
@@ -0,0 +1,30 @@
+# Backstage documentation
+# https://backstage.io/docs/features/software-catalog/descriptor-format/
+
+# !!! WARNING !!!
+# This is a template file with a number of fields that need to be filled before merging this to the default branch
+apiVersion: backstage.io/v1alpha1
+# Component, API, Template, Group, User, Resource, System, Domain, Location
+kind: unknown
```

Comment:
update this to Component or System. Turns out not using a built-in Kind value breaks the ingestion.